### ISSUE TITLE
Samsung remote: image not rendered 

### ISSUE THREAD

**Issue by XalaTheShepard** [dimagoltsman/generic-remote-control-card#28]
Dear all,

After the 2.8 update, the image of the Samsung remote is not rendered correctly.
Only the white markings of the buttons are visible.
Please find enclosed a screenshot.
I am running HA  Core 2022.7.0 in VM.

Many thanks for looking into this issue.

Kind regards,

Robin
![Screenshot_20220707_215929_io homeassistant companion android](https://user-images.githubusercontent.com/69390987/177862008-3add3278-4d24-414c-bd99-08a80eae9363.jpg)

**Comment by dimagoltsman** [comment]
there was an update, can u update and try again?

**Comment by XalaTheShepard** [comment]
I have reinstalled the latest version. In my Google Chrome browser on my pc the image is rendered correctly. In my HA app on my Android phone it is still the same issue as the image above. I have already cleared the cache of the app, but this brought no solution. 

Edit: I noticed on other browsers on my pc that the image is also not rendered. Clearing the cache of my Google Chrome on my pc revealed that the image is nor in the app nor in any browser rendered.
